common/lib/dependabot/shared_helpers.rb

```diff
@@ -25,14 +26,19 @@ module SharedHelpers
 
     def self.in_a_temporary_repo_directory(directory = "/",
                                            repo_contents_path = nil,
+                                           **options,
```

Comment:
❓ Should we avoid changing the method signature of this class?

It doesn't look like we are actually setting `options[:memo]` currently and the workspace tolerates this being `nil` by using placeholders.

I think we should ideally try to pass more meaningful strings in eventually so there's no need to change the use of `memo` inside the `Dependabot::Workspace`, but perhaps we should remove it from here for now until we have concrete plans to pass in some information about the change from the `FileUpdater`?

Reply:
That makes sense. Let's get things working and we can figure out how to pass context later.